docs/user_guide/mitigation.rst

```diff
@@ -68,6 +68,216 @@ Postprocessing
+        "Female": {
+            "p_ignore": 0.13186000824177083,
+            "prediction_constant": 0.035,
+            "p0": 0.9943698649710652,
+            "operation0": "[>0.5]",
+            "p1": 0.005630135028934835,
+            "operation1": "[>-inf]"
+        },
+        "Male": {
+            "p_ignore": 0.0,
+            "prediction_constant": 0.035,
+            "p0": 0.9999261555292187,
+            "operation0": "[>0.5]",
+            "p1": 7.384447078129242e-05,
+            "operation1": "[>-inf]"
+        }
+    }
+    >>> plot_threshold_optimizer(threshold_optimizer)
+
+The printed thresholding rules influence the predictions, so we need to first
+understand the formula that puts all the parts together.
+Instead of selecting a particular thresholding rule we can actually select
+any point between any two thresholding rules.
+This is a very important insight and occurs in almost every case.
+To reach such a point we will have to create the linear combination of the two
+rules as
+:math:`p_0 \cdot \text{operation}_0(\text{score}) + p_1 \cdot \text{operation}_1(\text{score})`.
+The goal in this case with equalized odds is to reach equal true and false
+positive rates, which are conveniently plotted on the y- and x-axes.
+It follows that they'd be equal at any spot where the two curves intersect.
+We can relax this a little bit since in many cases one lies strictly above
+the other, just like in this example.
+Imagine the diagonal of the ROC plot from :math:`(0,0)` to :math:`(1,1)`.
+It is usually not considered as useful since we could achieve the same result
+with random classifiers with probabilities to get label 1 ranging from 0 to 1.
+If we have one curve strictly below the other, we can still get their true and
+false positive rates to match by "pulling" the higher one towards the
+diagonal. In reality we consider the overlap, so only the points on the lower
+curve are available to us. From those points we choose the one with the best
+value in terms of the overall objective (not per group).
+
+Mathematically, the following formula represents this interpretation of the
+chart, where :math:`c` represents the `prediction_constant` from the printed
+dictionary above.
+
+.. math::
+
+    p_{ignore} \cdot c + (1-p_{ignore}) \cdot \left(p_0 \cdot \text{operation}_0(\text{score}) + p_1 \cdot \text{operation}_1(\text{score})\right)
+
+In other words, we create a linear combination of the binary thresholding rule
+outputs. :math:`p_0` and :math:`p_1` indicate how close we are to the points
+representing the two thresholding rules, and :math:`p_0+p_1=1`. `p_ignore` is
+:math:`0` if the curve does not need to be "pulled" towards the diagonal,
+i.e., when the selected solution lies on the curve itself as opposed to below.
+Above, the curve for "Male" is strictly below the curve for "Female", so
+:math:`p_{ignore}=0` for "Male" and non-zero for "Female".
+
+Importantly, :math:`p_{ignore}` is only required for equalized odds. Equalized
```

Comment:
Don't you need an \mbox or something equivalent for the subscripted text? Otherwise it will be rendered as algebra?